templates/README.tpl.md

```diff
@@ -0,0 +1,169 @@
+
+- `<APP_NAME>`: the application name
+- `<APP_REPO>`: Github app repository name
+- `<APP_SHORT_DESCRIPTION>`: a quick app description
+- `<APP_PORT>`: app running port
+- `<APP_MAINTAINER>`: Github main maintainer username (don't forget `@` :))
+- `<SLUG_TX>`: transifex app slug
+- `<SLUG_GH>`: Github repository slugname
+- `<SLUG_NPM>`: NPM slugname
+
+---
+
+![Travis build status shield](https://img.shields.io/travis/cozy/<APP_REPO>.svg)
+![NPM release version shield](https://img.shields.io/npm/v/<SLUG_NPM>.svg)
+![Github Release version shield](https://img.shields.io/github/release/cozy/<APP_REPO>.svg)
+![NPM Licence shield](https://img.shields.io/npm/l/<SLUG_NPM>.svg)
+
+
+[Cozy][cozy] <APP_NAME>
+=======================
+
+
+What's Cozy?
+------------
+
+![Cozy Logo](https://raw.githubusercontent.com/cozy/cozy-guidelines/master/templates/cozy_logo_small.svg)
+
+[Cozy][cozy] is a platform that brings all your web services in the same private space.  With it, your webapps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one's tracking you.
+
+
+What's <APP_NAME>?
+------------------
+
+<APP_SHORT_DESCRIPTION>
+
+
+Hack
+----
+
+### Install and run in dev mode
+
+Hacking the <APP_NAME> app requires you to [setup a dev environment][setup].
+
+You can then clone the app repository and install dependencies:
+
+```sh
+$ git clone https://github.com/cozy/<SLUG_GH>.git
+$ cd <SLUG_GH>
+$ npm install
+```
+
+:pushpin: If you use a node environment wrapper like [nvm] or [ndenv], don't forget to set your local node version before doing a `npm install`.
+
+Cozy's apps use a standard set of _npm scripts_ to run common tasks. You can so start you development workflow with:
+
+```sh
+$ cd <SLUG_GH>
+$ npm run watch
+```
+
+and point your browser to http://localhost:<APP_PORT>.
+
+
+### Run it inside the VM
+
+You can easily view your current running app in your VM, use [cozy-dev]:
+
+```sh
+# in a terminal, run your app in watch mode
+$ cd <SLUG_GH>
+$ npm run watch
+```
+
+```sh
+# in another terminal, install cozy-dev (first time) and run the deploy
+$ cd <SLUG_GH>
+$ npm install -g cozy-dev
+$ cozy-dev deploy <APP_PORT>
+```
+
+your app is available in your vm dashboard at http://localhost:9104.
+
+
+### Tests
+
+Tests are run by [mocha] under the hood, and written using [chai] and [sinon]. You can easily run the tests suite with:
+
+```sh
+$ cd <SLUG_GH>
+$ npm run test
+```
+
+:pushpin: Don't forget to update / create new tests when you contribute to code to keep the app the consistent.
+
+
+### Resources
+
+All documentation is located in the `/docs` app directory. It provides an exhaustive documentation about workflows (installation, development, pull-requests…), architecture, code consistency, data structures, dependencies, and more.
+
+Feel free to read it and fix / update it if needed, all comments and feedback to improve it are welcome!
+
+
+### Open a Pull-Request
```

Comment:
I'm currenlty adding a `CONTRIBUTING.md` doc, I link it into the README for this PR.